cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -89,15 +89,16 @@ type CMServer struct {
 	ServiceAccountKeyFile             string
 	RootCAFile                        string
 
-	ClusterName                   string
-	ClusterCIDR                   net.IPNet
-	AllocateNodeCIDRs             bool
-	EnableProfiling               bool
-	EnableHorizontalPodAutoscaler bool
-	EnableDeploymentController    bool
-
-	Master     string
-	Kubeconfig string
+	ClusterName                       string
+	ClusterCIDR                       net.IPNet
+	AllocateNodeCIDRs                 bool
+	EnableProfiling                   bool
+	EnableHorizontalPodAutoscaler     bool
+	EnableDeploymentController        bool
+	EnableHostPathProvisioner         bool
```

Comment:
It needs to be default off.  There's also a "cloud must be nil" check in the code to restrict this further.